azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeProviderMetadata.java

```diff
@@ -71,8 +68,10 @@ public static Properties defaultProperties() {
       properties.put(CREDENTIAL_TYPE, CLIENT_CREDENTIALS_SECRET.toString());
       properties.put(RESOURCE_GROUP_NAME, "jcloudsgroup");
       properties.put(IMAGE_PUBLISHERS, "Canonical,RedHat");
-      properties.put(DEFAULT_IMAGE_LOGIN, "jclouds:Password1!");
       properties.put(TIMEOUT_NODE_TERMINATED, 60 * 10 * 1000);
+      // Default credentials for all images
+      properties.put("azurecompute-arm.image.login-user", "jclouds:Password1!");
+      properties.put("azurecompute-arm.image.authenticate-sudo", "true");
```

Comment:
Using this properties, jclouds will automatically populate these credentials as the Image default credentials for each image. No need for a custom `PopulateFoo` class to do that. Users will also have an easy way to override these defaults by overriding the properties.